sys/socket_packet.txt

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# http://man7.org/linux/man-pages/man7/packet.7.html
+# https://www.kernel.org/doc/Documentation/networking/packet_mmap.txt
+
+# AF_PACKET support.
+
+include <linux/socket.h>
+include <linux/net.h>
+include <linux/if_ether.h>
+include <linux/if_arp.h>
+
+resource sock_packet[sock]
+
+define ETH_P_ALL_BE	htons(ETH_P_ALL)
+
+socket$packet(domain const[AF_PACKET], type flags[packet_socket_type], proto const[ETH_P_ALL_BE]) sock_packet
```

Comment:
I would do it as `const[ETH_P_ALL, int16be]`. Defines are kind of red flag meaning that we can't handle something, but we actually can handle this one.

Reply:
Done. We actually couldn't handle this, but we can now (added another trivial commit).

Reply:
Reverted as per offline discussion.